src/iqm_client/iqm_client.py

```diff
@@ -203,24 +207,36 @@ class SingleQubitMapping(BaseModel):
     'physical qubit name'
 
 
-def serialize_qubit_mapping(qubit_mapping: dict[str, str]) -> list[SingleQubitMapping]:
-    """Serializes a qubit mapping dict into the corresponding IQM data transfer format.
+QubitMapping = list[SingleQubitMapping]
+"""Type that represents qubit mapping for a circuit, i.e. a list of single qubit mappings
+for all qubits in the circuit."""
+
+
+BatchQubitMapping = list[QubitMapping]
+"""Type that represents list of qubit mappings, one per each circuit in a circuit batch"""
+
+
+def serialize_qubit_mappings(qubit_mappings: list[dict[str, str]]) -> BatchQubitMapping:
+    """Serializes a list of qubit mapping dicts into the corresponding IQM data transfer format.
```

Comment:
```suggestion
    """Serializes a list of qubit mappings into the corresponding IQM data transfer format.
```
Generally speaking, type names should not be repeated in the docstrings if possible, since they can be seen in the type annotations.